Tools/ardupilotwaf/chibios.py

```diff
@@ -383,6 +383,8 @@ def generate_hwdef_h(env):
         os.mkdir(hwdef_out)
     python = sys.executable
     cmd = "{0} '{1}' -D '{2}' '{3}' {4} --params '{5}'".format(python, hwdef_script, hwdef_out, env.HWDEF, env.BOOTLOADER_OPTION, env.DEFAULT_PARAMETERS)
+    if env.HWDEF_EXTRA:
```

Comment:
I'd prefer the nargs='+' option which means you'd just add the extra filename on the end here